AgentBlob/GenerateAgentBlob.ps1

```diff
@@ -0,0 +1,79 @@
+﻿
+#$BootstrapUrl = "http://52.151.39.54/performance"
+
+function Download-7Zip {
+    Write-output "Download-7Zip"
+    $urifile = "https://7-zip.org/a/7z1801-x64.msi"
```

Comment:
We should use CDN instead of 3rd party URLs. Please send me all such URLs and I will copy them to CDN

Reply:
I would say we can do the same for the `devcon.exe` binary.

Therefore, we can remove this block of code with extracting a `cab` file, and simply fetch `devcon.exe` from CDN.

Reply:
Yeah, we want to move them to CDN after we got thing fully validated.